lib/App/DuckPAN/Query.pm

```diff
@@ -2,95 +2,125 @@ package App::DuckPAN::Query;
 # ABSTRACT: Main application/loop for duckpan query
 
 use Moo;
-
-my $query;
-my $history_path;
-
+use DDG;
+use DDG::Request;
```

Comment:
It seems that moving this here has cause all the Travis tests to fail because DDG::Request cannot be found in @INC. The problem with DDG is that it's not a CPAN Repository, instead its hosted on our own duckpan.org.

I'm not entirely sure how/why the `require...import` method makes a difference with regards to making DDG::Request accessible...